docs/_config.yml

```diff
@@ -18,7 +18,7 @@ gh_edit_branch: master
 gh_edit_source: docs
 gh_edit_view_mode: edit
 
-logo: "resources/hyland-alfresco-logos.png"
+logo: "images/hyland-alfresco-logos.png"
 favicon_ico: "favicon.ico"
```

Comment:
can you add also the favicon which is missing as well? 🤒 

https://github.com/Alfresco/alfresco-helm-charts/blob/OPSEXP-2620-jekyll/docs/images/favicon.ico